sysl2/sysl/parse/tests/foreign_import_swagger.sysl

```diff
@@ -0,0 +1,20 @@
+import foreign_import_swagger.yaml as pck.app ~swagger
```

Comment:
What's the significance of `pck.app`? Should the ref below be `…depth=pck.app.SimpleObj&…` or similar? Referencing syntax and semantics is something we need to clean up sooner rather than later.

Reply:
the import code needs an application name (at least for swagger imports anyway), so in that line `app` would be the application name for use later. `pck` would be the full package name, maybe the example should be something better like `com.anz.dependancy.foo` where `foo` ends up being the app name as the rest is the package name.